### ISSUE TITLE
moqup

### ISSUE THREAD

**Issue by MsLobster** [dbertella/food-and-quote#11]
![screen shot 2017-03-08 at 23 20 16](https://cloud.githubusercontent.com/assets/26073839/23728638/d363d9a2-0455-11e7-9e7f-605416632c2e.png)

**Comment by MsLobster** [comment]
@buseca we need an opinion from a good UX designer do you know somebody?

**Comment by buseca** [comment]
Yea for sure, i'll give you his mail address,
but in the meanwhile:
I would go for faster interactions, moving the check box and radio button in the results page, using them as filters on the already showed content. So you show content to the user in just one step.
I would also consider to let user adding more ingredients in the first page; you can check the "tags" pattern you find on Tumblr or other blog platform, which is super fast and have auto complete.
Let's consider also adding a random recipe button, for "easy exploring" purpose.
:-)

**Comment by dbertella** [comment]
Thanks, I'm not sure though, the first page doesn't have checkboxes and filter but just the search field, so are you suggesting to have just one page with the search and then add the filters directly once we show the content right?

> Let's consider also adding a random recipe button, for "easy exploring" purpose.
I like the random recipe button

**Comment by MsLobster** [comment]
Yeaaah I always wanted a shake-your-phone-get-a-random-recipe!!! But a "surprise me" button can be fine too. :)
Not sure about what you say about the filters @buseca though. I like the possibility to choose these options (the kind of dish, type of diet) before to search. 
A good compromise would be a fixed filter button (see screenshots) placed in the upper right corner. It would be seen on a new page though so you can have a clear look at what to filter or not.
![screenshot_2017-03-09-12-32-42-751_com aufeminin marmiton activities](https://cloud.githubusercontent.com/assets/26073839/23750652/de52bf16-04c5-11e7-88bd-9c0b19ab7612.png)
![screenshot_2017-03-09-12-33-05-866_com aufeminin marmiton activities](https://cloud.githubusercontent.com/assets/26073839/23750653/de571bd8-04c5-11e7-8e4a-381e3b6a7763.png)

**Comment by buseca** [comment]
Yes, I think you should be able to see recipes yet in the second page, and
then you can filter them.
It would be great to have at least one recipe thumbnail also in the first
page , just to create context.
Be aware with the number of categories, if you are not planning to fill it
with 100 recipes, 7 categories are a lot. Try to use few of them.
And maybe you don't need both filters and categories. I this this kind of
research should be more explorative and "visual" based.

Il 09 mar 2017 1:44 PM, "MsLobster" <notifications@github.com> ha scritto:
